pkg/bus/bus_test.go

```diff
@@ -64,9 +64,9 @@ func TestQueryHandlerReturnsError(t *testing.T) {
 	err := bus.Dispatch(&testQuery{})
 
 	if err == nil {
-		t.Fatal("Send query failed " + err.Error())
+		t.Fatal("Send query failed")
```

Comment:
looking at the log message it seems like this error handling is invalid. 
logging `Send query failed` when err is nil?

Reply:
@bergquist well, this is, I guess, reverse logic inside a test case. The problem was trying to get a string from `nil`.